README.md

```diff
@@ -83,24 +82,27 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://fascalfocus1.onrender.com/)
+- [Video link](https://www.loom.com/share/0aeef1725c9b41b8999d66bb4da48ab9)
```

Comment:
- [ ] The video should contain a highlight about a piece of the code you find interesting or are proud of making.

Reply:
- Check `3.20` upward for highlights as i stated in the vidoe